src/background/modules/Detect/Pleroma.js

```diff
@@ -84,13 +86,16 @@ export function getTabToModify(requestDetails) {
 }
 
 /**
- * Find the follow URL.
+ * Find the status URL.
  *
  * @public
+ * @param {URL} url
  * @returns {Promise}
  */
-export function getTootUrl() {
-    throw new NotSupportedError("getTootUrl() is not supported");
+export function getTootUrl(url) {
+    return new Promise((resolve, reject) => {
+        resolve("https://" + url.host + "/notice/" + url.searchParams.get("status_id"));
```

Comment:
```suggestion
        resolve(`https://${url.host}/notice/{url.searchParams.get("status_id")}`);
```

Let's use string interpolation here, that's more modern…